packages/core/src/routes/applications/application.ts

```diff
@@ -168,6 +166,12 @@ export default function applicationRoutes<T extends AuthedRouter>(
         ...rest,
       });
 
+      if (application.type === ApplicationType.Protected) {
+        await protectedApps.syncAppConfigsToRemote(application.id);
```

Comment:
Should update the API's response status guard to include the new added error status codes?

Reply:
updated